tool-bar.ios.ts

```diff
@@ -111,7 +111,7 @@ export class ToolBar extends ToolBarDefinition {
         let navBarHeight = 0;
 
         if (heightMode != layout.EXACTLY) {
-            let toolbarSize = this._ios.intrinsicContentSize();
+            let toolbarSize = this._ios.intrinsicContentSize;
             newHeight = toolbarSize.height;
```

Comment:
http://docs.nativescript.org/api-reference/modules/_utils_utils_.ios.html#getter

This could be used to retain backwards compatibility while supporting the latest version.